docs/glossary.md

```diff
@@ -0,0 +1,24 @@
+---
+id: Glossary
+title: Backstage Glossary
+description:
+  List of all abbreviations, terms, phrases used in Backstage together with its
+  explainations.
```

Comment:
Annoying SEO hack for Docusaurus. Ref: https://github.com/backstage/backstage/pull/3927

```suggestion
# prettier-ignore
description: List of all abbreviations, terms, phrases used in Backstage together with its explanations.
```